R/stao/CFEC_BIT_processing_199.R

```diff
@@ -0,0 +1,67 @@
+################
+#Issue #199: CFEC Basic Information Table (BIT)
+#Data Processing
+#February 2018
+#Sophia Tao
+################
+
+
+
+# read in
+bit <- read.csv("/home/stao/my-sasap/199_CFEC/BIT.csv", header = T)
+
+# delete unnecessary column
+bit$X....Preliminary <- NULL
+
+# replace "." with "NA"
+bit$Average.Permit.Price[bit$Average.Permit.Price=="."] <- NA
```

Comment:
If you want to do this for the whole data frame, here's an alternative: `df[df=="."] <- NA` https://stackoverflow.com/questions/19503266/replace-all-particular-values-in-a-data-frame

Reply:
also, to make code more readable, it's generally good to put spaces around `=` or `==` (which I didn't do, oops!) http://style.tidyverse.org/